src/main/java/com/revature/services/JMeterService.java

```diff
@@ -39,14 +48,34 @@
 
     // replace user with username later
     public static final String BASE_FILE_PATH = "./datafiles/user_";
-    
-    public static final String PROPERTIES_PATH = "src/main/resources/jmeter.properties";
+
+    public static final String PROPERTIES_PATH = "/jmeter.properties";
 
     private LoadTestConfig testConfig = new LoadTestConfig();
-    
+
     @Autowired
     private SwaggerSummaryService sss;
 
+    @Autowired
+    private ResultSummaryCsvService rscs;
+
+    @PostConstruct
+    private void loadProperties() {
+        InputStream inputStream = getClass().getResourceAsStream(PROPERTIES_PATH);
+        Properties p = new Properties();
+        
+        try {
+            Field field = JMeterUtils.class.getDeclaredField("appProperties");
+            field.setAccessible(true);
+            p.load(inputStream);
+            field.set(this, p);
```

Comment:
Reflection was seen as the best option here because the JMeterUtils methods were insufficient for loading in properties without using filepaths, which would have posed a problem for loading the jmeter.properties resource within a .jar